migration-guide.adoc

```diff
@@ -48,3 +48,11 @@ MyEntity myEntity = session.find(MyEntity.class, 2, LockMode.WRITE);
 ```
 
 only the entity with id equals to 2 will be loaded but the proxy will not be initialized.
+
+[[metamodel-generation]]
+== Integrating Static Metamodel Generation
+The integration of static metamodel generation in a project has changed; the recommended way to do this now is by harnessing the annotation processor classpath. This is true for both gradle and maven.
```

Comment:
Minor...

Other than this, looks great

```suggestion
The integration of static metamodel generation in a project has changed; the recommended way to do this now is by harnessing the annotation processor classpath. This is true for both gradle and maven.
```

This is true for both Gradle and Maven.